hazelcast/src/main/java/com/hazelcast/instance/DefaultNodeContext.java

```diff
@@ -52,4 +51,21 @@ public ConnectionManager createConnectionManager(Node node, ServerSocketChannel
                 node.nodeEngine.getMetricsRegistry(),
                 threadingModel);
     }
+
+    private TcpIpConnectionThreadingModel createTcpIpConnectionThreadingModel(Node node, NodeIOService ioService) {
+        boolean spinning = Boolean.getBoolean("hazelcast.io.spinning");
```

Comment:
this is ambiguous as you can spin on socket channels and you can also spin on selectors.